runner/daemon/src/mill/daemon/MillCliConfig.scala

```diff
@@ -97,6 +97,11 @@ case class MillCliConfig(
       doc = """Watch and re-run the given tasks when when their inputs change."""
     )
     watch: Flag = Flag(),
+    @arg(
+      name = "watch-via-fs-notify",
```

Comment:
Let's shorten this to `--notify-watch` here and in the other PR, the current name is a bit of a mouthful